src/renderers/WebGLRenderer.js

```diff
@@ -2803,6 +2803,8 @@ function WebGLRenderer( parameters ) {
 
 	this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level ) {
 
+		level = level || 0;
```

Comment:
I guess this makes it more clear?
```js
if ( level === undefined ) level = 0;
```

Reply:
Will change to `if ( level === undefined ) level = 0;`